virus/RedMiteAssociatedHypovirus1-virTemplate.md

```diff
@@ -0,0 +1,117 @@
+
+An important finding across all datasets was the identification of RdRP enzymes, which play a crucial role in viral replication. As a result of their widespread presence across different host species, Red mite associated hypovirus 1 must be able to adapt to diverse ecological conditions. The three data sets consisting of fungi, human skin cells, and plant tissues contain this viral genetic information indicating that this virus is highly adaptable thus allowing its spread through species. A wide variety of ecosystems are also covered by the datasets. Liu et al. focused their study on fungi which are grown in decayed wood ecosystems. Torish et al. focused on human skin microbiomes and Gu et al. focused on plant pathology. This virus has shown potential of thriving under a wide range of environmental conditions [Elena et al., 2009]. It is possible that multiple host switching events have occurred during the evolution of this virus, as evidenced by the diversity of hosts, ranging from fungi to humans and plants [Elena et al., 2009]. Most likely, this virus's ability to adapt to diverse biological systems is a result of its genetic makeup [Elena et al., 2009]. 
+
+
+### Virus Genome {Q3}
+
+```{r}
+library(ggplot2)
+library(gggenes)
+example_genes <- data.frame(
+  gene = c("polyprotein", "RNA_dep_RNAP" ),
+  start = c(39, 283 ), 
+  end = c(427, 676 ),
+  strand = c("+", "+"),
+  feature = c("ORF", "ORF")
+)
+ggplot(example_genes, aes(xmin = start, xmax = end, y = gene, fill = gene)) +
+  geom_gene_arrow() +
+  scale_fill_brewer(palette = "Set3") +
+  theme_minimal() +
+  xlab("Position") +
+  ylab("Gene")
+
+```
+Figure: This figure depicts the ORF for RNA dependent RNAP and polyprotein. Both the ORF have have partial CNS. The start and stop locations for the ORF were collected from ORFfinder from blast. The polyprotein ORF had the amino acid code of MEDRQNDQNDDVFSFFSPSFSAATPSTLFNRSAYSSSSSSGDDESQPSVDDSNKRIDYMIQFLDRRLSEDGNHDGIGDGNGSDSLPEFVGKCGESGIFKVPIRSAVHPNRPPSLDVRPHPLRETQIGRFLRTMTSTERQLWTGGEDGALRVWEFSELYGS. There are several motifs in the catalytic center that contain a number of conserved amino acid residues[Tirosh et al., 2018]. Viruses that hold this strand of polyproteins usually are known to be a positive-strand RNA eukaryotic virus[Tirosh et al., 2018]. A polyprotein consists of chains of smaller proteins[Crépin  et al., 2015]. This mecanism was developed to organize virus proteomes[Crépin  et al., 2015]. 
```

Comment:
Add Figure number. For amino acid codes please use a code-block to distinguish it from written text `like this`.